ansible/jonzeolla/labs/roles/cloud9/files/check-partition.sh

```diff
@@ -18,3 +18,11 @@ if [[ "${actual_size}" -lt "${desired_size}" ]]; then
   echo "Disk ${primary_disk} is too small: ${actual_size}B < ${desired_size}B"
   exit 1
 fi
+
+partition_name="$(lsblk -o NAME,SIZE,MOUNTPOINT | awk '$NF == "/" {print $1}' | sed 's/^[^a-zA-Z0-9]*//')"
+partition_size="$(lsblk -o NAME,SIZE,MOUNTPOINT | awk '$NF == "/" {print $2}')"
+
+if [[ "${partition_size}" != "${desired_size_in_gb}G" ]]; then
```

Comment:
Without this addition, when the disk resize was successful but the partition resize wasn't, it succeeded. That wasn't the intent of this validation, so this improves the check